### PR TITLE
Cleanup old deps that is not needed

### PR DESCRIPTION
**Pull request by mariogrip**
Seems to be not needed anymore as we currently does not even install the platform apis nor do we use gsettings

@bhush9

### CODE REVIEW

**Comment by bhush9** [comment]
Checks are failing but this looks good to me.

File: libtelephonyservice/applicationutils.cpp [hunk 1]
```diff
@@ -66,7 +56,7 @@ bool ApplicationUtils::checkApplicationRunning(const QString &serviceName)
 bool ApplicationUtils::openUrl(const QUrl &url)
 {
     if (qgetenv("TELEPHONY_SERVICE_TEST").isEmpty()) {
-        url_dispatch_send(url.toString().toUtf8().constData(), urlDispatchCallback, 0);
+        QDesktopServices::openUrl(url);
```
Comment (UniversalSuperBox) [inline libtelephonyservice/applicationutils.cpp]: No apps open from the indicator with this patch installed.

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -64,16 +65,26 @@ find_package(PkgConfig REQUIRED)
 pkg_check_modules(TP_QT5 REQUIRED TelepathyQt5)
 pkg_check_modules(TP_QT5_FS REQUIRED TelepathyQt5Farstream)
 pkg_check_modules(NOTIFY REQUIRED libnotify)
-pkg_check_modules(MESSAGING_MENU REQUIRED messaging-menu)
 pkg_check_modules(UserMetrics REQUIRED libusermetricsinput-1)
```
Comment (bhush9) [inline CMakeLists.txt]: This should  removed from here.

**Review by UniversalSuperBox — APPROVED** [review]
Messaging menu, with direct replies, tests fine here.

**Pull request merged**